internal/controllers/s3userclaim/provisioner.go

```diff
@@ -131,6 +133,81 @@ func (r *Reconciler) ensureReadonlySubuser(ctx context.Context) (*ctrl.Result, e
 	return subreconciler.ContinueReconciling()
 }
 
+func (r *Reconciler) ensureOtherSubusers(ctx context.Context) (*ctrl.Result, error) {
+	specSubUsers := r.s3UserClaim.Spec.SubUsers
```

Comment:
Instead of `subUsers` and `specSubUsers` I recommend `desiredSubUsers` and `currentSubUsers`.

Reply:
I changed `specSubUsers` to desiredSubUsers and `cephSubUser` to `currentSubUser`.